specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1434,6 +1434,10 @@
           "description": "The recipient Email Address",
           "type": "string"
         },
+        "internetMessageId": {
```

Comment:
Is this a uuid? Is this value exposed in the client library as a Guid or a string?

Reply:
This is not a uuid. it starts with a "<" and end with a ">"
example "\<xyc.abc.123@contoso.com\>"